chia/daemon/server.py

```diff
@@ -1181,15 +1176,15 @@ def daemon_launch_lock_path(root_path: Path) -> Path:
     A path to a file that is lock when a daemon is launching but not yet started.
     This prevents multiple instances from launching.
     """
-    return root_path / "run" / "start-daemon.launching"
```

Comment:
the name `.launching` here suggests that this lock file was intended to only be used while starting the daemon

Reply:
Yeah sure but whats the point? And whats wrong with keeping the lock while it's running? @richardkiss can you explain the intention since you implemented this here https://github.com/Chia-Network/chia-blockchain/pull/217?

Reply:
I think maybe the idea was, using unix sockets, you could delete the unix socket and the daemon would notice and gracefully shut down. (Obviously that doesn't work anymore since you can't delete a tcp socket.)